posthog/hogql/database/schema/util/where_clause_extractor.py

```diff
@@ -459,6 +454,9 @@ def visit_alias(self, node: ast.Alias) -> bool:
     def visit_tuple(self, node: ast.Tuple) -> bool:
         return all(self.visit(arg) for arg in node.exprs)
 
+    def visit_array(self, node: ast.Tuple) -> bool:
```

Comment:
@mariusandra this was a pretty annoying miss, is there a better way rather than having to add all possible types here manually?

Reply:
Hmm, normally the visitors inherit from `CloningVisitor` or `TraversingVisitor`, which have defaults for all the different nodes, but this one seems to derive from just `Visitor`. I'm not exactly sure why is that... probably because it returns booleans. There's a way to just define a `def visit(self, node)` method and return false if calling something not found. Otherwise, someone will indeed have to add all the different nodes here...